### PR TITLE
Feature/feedback form validation

### PR DESCRIPTION
**Pull request by Ksenia-Tulikova**
Add form validation using formSchema validation.
I've read that Joi is a powerfull validator wich provides validation schemas for forms, so i decided start using it on my project.
Also i moved all thunks to another file.
PREVIEW: 
image

### CODE REVIEW

File: client/webpack/paths.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ const path = require('path');
 module.exports = {
   root: path.resolve(__dirname, '../'),
   outputPath: path.resolve(__dirname, '../', 'build'),
-  entryPath: path.resolve(__dirname, '../', 'src/index.js'),
+  entryPath: path.resolve(__dirname, '../', 'src/thunks.js'),
```
Comment (AV29) [inline client/webpack/paths.js]: not sure about this change... why did you need to change entry point?

File: client/src/modules/Feedback/actions/thunks.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import { addFeedback } from '../../../feedbackManager';
+import { getFormData } from '../selectors';
+import { validation_rules } from '../validation/rules';
```
Comment (AV29) [inline client/src/modules/Feedback/actions/thunks.js]: 'validation_rules' Is it a consistent kind of naming?

Comment (Ksenia-Tulikova) [reply client/src/modules/Feedback/actions/thunks.js]: Ohhh, ill change it on 'rules'

File: client/webpack/paths.js (hunk 1 above)
Comment (Ksenia-Tulikova) [reply client/webpack/paths.js]: My fault, i forgot to change it back:(

**Review by AV29 — APPROVED** [review]
